client/src/containers/App/routes/index.js

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
 import { Route, Switch, Redirect } from 'react-router-dom';
 
-// components
+// components public
 import Signup from '../../Signup/Signup';
 import Login from '../../Login/Login';
 import Events from '../../Events/Events';
 import ForgotPassword from '../../ForgotPassword/ForgotPassword';
-import PublicRoutes from './public.routes';
+import Organisation from '../../Organisation/Organisation';
 import PrivateRoutes from './private.routes';
```

Comment:
please remove PrivateRoutes import from here. You can add comment above line number 4 like // privete routes and then define import PrivateRoutes from './private.routes';

Reply:
done!